dbms/programs/client/Client.cpp

```diff
@@ -1672,7 +1664,7 @@ class Client : public Poco::Util::Application
               * the "\n" is used to distinguish this case because there is hardly a chance an user would use "\n"
               * as the password.
               */
-            ("password", po::value<std::string>()->implicit_value("\n"), "password")
+            ("password", po::value<std::string>()->implicit_value("\n","\\n"), "password")
```

Comment:
?

Reply:
When default arg was printed it destructed the option list with new line, now it doesn't.

Reply:
Ok. But this code is wrong nevertheless, because it doesn't contain explanation in comments.

Reply:
But default password is not a line feed. It is some ugly trick that @proller managed to put in code.